lib/modules/reducer.js

```diff
@@ -35,6 +46,7 @@ export default (state=DEFAULT, action={}) => {
         currentPage: pageData,
       };
     }
+
```

Comment:
🔕  do we have any rules in the styleguide regarding spaces between case statements?